.github/workflows/deploy-main.yml

```diff
@@ -62,6 +62,7 @@ jobs:
           ANALYTICS_FE_KEY=${{ secrets.ANALYTICS_FE_KEY }}
           VERSION=sha-${{ steps.vars.outputs.sha_short }}
           TRACETEST_ENV=main
+          PUBLIC_PATH=/
```

Comment:
This will be applied to `beta.tracetest.io` and our integration instance. To apply this change to demo, you have to edit the file: workflows/release-server.yml instead

Reply:
@mathnogueira added to release-server.yaml as well